evap/student/templates/student_vote.html

```diff
@@ -213,6 +268,19 @@ <h3 class="mb-3">{{ course.name }} ({{ course.semester.name }})</h3>
                     presaveButton.text(originalPresaveText);
                 }, 2500);
             }
+
+            // handle text_results_publish_confirmation checkbox changes
+            function textResultsPublishConfirmationTopChanged() {
+                $("#text_results_publish_confirmation_bottom").prop('checked', $("#text_results_publish_confirmation_top").prop('checked'));
+                if($("#text_results_publish_confirmation_top").prop('checked')) {
+                    $("#bottom_text_results_publish_confirmation_card").hide();
+                } else {
+                    $("#bottom_text_results_publish_confirmation_card").show();
+                }
```

Comment:
`$("#bottom_text_results_publish_confirmation_card").toggle($("#text_results_publish_confirmation_top").prop('checked'))`?
or maybe put the top prop into a variable